src/content/docs/configuration/providers.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Providers
+description: Understand Providers and configure new Targets.
+sidebar:
+  label: Providers
+---
+
+Daytona abstracts the management and deployment of Workspaces into 2 related concepts:
+
+1.  __Provider__  
+    A platform capable of deploying Workspaces on.
+    Examples of Providers may include:
+    *   Container management platforms such as Docker and Podman;
+    *   Cloud hosting providers such as AWS or DigitalOcean.
+2.  __Target__  
+    A set of configuration that governs how Daytona manages and deploys Workspaces.
+    Each Target is tied to an individual Provider.
+    Daytona allows you to configure and switch between multiple Targets at Workspace creation time.
```

Comment:
Suggestion: `Daytona allows you to choose between multiple Targets at Workspace creation time.`

`configure` is wrong because you can't configure new targets when creating a workspace (that's done separately) and `switch` makes it seems like you activate the target or switch the default.